### PR TITLE
rtmp-services: Add CAM4 streaming service

### PR DESCRIPTION
**Pull request by codertux**
CAM4 is an adult webcam community.

http://www.cam4.com

### CODE REVIEW

**Comment by ONSBalder** [comment]
Can one of the admins verify this patch?

**Comment by jumoog** [comment]
tune parameter is usefull

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -709,6 +709,40 @@
             "recommended": {
                 "max video bitrate": 3500
             }
+        },
+        {
+            "name": "CAM4",
+            "common": true,
```
Comment (jumoog) [inline plugins/rtmp-services/data/services.json]: please remove common see -> https://github.com/jp9000/obs-studio/pull/577#issuecomment-234386514

**Comment by robske110** [comment]
I think you should meld 3596286 into e56fcdc.

**Comment by codertux** [comment]
Hello, just fixed the merge conflicts, can someone take a look at it again please?

**Pull request closed without merging**